src/templates/partials/header.njk

```diff
@@ -1,10 +1,25 @@
-<header>
-    <h1>Site Header</h1>
-    <nav>
-        <ul>
-            <li><a href="/">Home</a></li>
-            <li><a href="/about">About</a></li>
-            <li><a href="/contact">Contact</a></li>
-        </ul>
-    </nav>
+<header class="header">
+  <a href="{{ baseUrl }}/index.html" class="logo">
+    <img src="{{ baseUrl }}/src/images/static/binabox-logo.webp" alt="Binabox Logo" class="logo__image" />
+  </a>
+  <nav class="main-nav">
+    <ul class="main-nav__list">
+      <li class="main-nav__list-item"><a class="main-nav__link" href="{{ baseUrl }}/index.html">Home</a></li>
+      <li class="main-nav__list-item"><a class="main-nav__link" href="{{ baseUrl }}/blog.html">Blog</a></li>
+      <li class="main-nav__list-item"><a class="main-nav__link" href="#">Catalog</a></li>
+      <li class="main-nav__list-item"><a class="main-nav__link" href="#">Collection</a></li>
+    </ul>
+    <div class="header__search">
+      <form action="/search" method="GET">
+        <input type="text" name="query" placeholder="Search" required aria-label="Search" class="search__input" />
```

Comment:
type = search

Reply:
you use element of a `search` block in class, but you don't have block class in html